### ISSUE TITLE
inherit config settings

### ISSUE THREAD

**Issue by BartvanMarrewijk** [Pointcept/Pointcept#183]
First thank you for this great github with all sota algorithms.
I am struggling with inheriting information from the .py files in the config.

For example, currently it is not possible to inherit the number of classes from the .py files in _base

```
_base_ = [
    "../../Pointcept/configs/_base_/default_runtime.py",
    "../load_data.py"]

import os
import json
# misc custom setting
batch_size = 4  # bs: total bs in all gpus
mix_prob = 0.8
empty_cache = False
enable_amp = True

grid_size = 0.002

# model settings
model = dict(
    type="DefaultSegmentor",
    backbone=dict(
        type="Swin3D-v1m1",
        in_channels=9,
        num_classes=len(_base_.classes),
etc
```

While running this piece of code I will get an error:
AttributeError: 'list' object has no attribute 'classes'

What am I doing wrong, is it possible to inherit information from the load_data.py file?

Thanks in advance

**Comment by Gofinge** [comment]
Hi, the config file is formatted as python code. I think it can not directly inherit information from _base_ like yaml file.

**Comment by BartvanMarrewijk** [comment]
Ah okay, thanks for the confirmation. I thought maybe it works the same as in the openmmlab library, which also uses this 
_base_ = [../your_data.py]
But for that library the abovementioned approach works.

**Comment by Gofinge** [comment]
Yes, it is the same config core as openmmlab library. The core supports both yaml and py and I choose to use py for configs in Pointcept. `_base_ = ["../_base_/default_runtime.py"]` works in py config as it also satisfies the grammar of Python files. Yet _base_.classes (List.classes) avoid the grammar of Python files. I think it is the reason why it doesn't work.
